Packages/Tracking/Core/Runtime/Scripts/Utils/AndroidServiceBinder.cs

```diff
@@ -0,0 +1,94 @@
+namespace Leap.Unity
+{
+    using System;
+    using UnityEngine;
+
+    public static class AndroidServiceBinder
+    {
+#if UNITY_ANDROID
```

Comment:
Why not `#if` the whole class (or indeed at this to an Android only asmdef?) Might be a cleaner solution?